autopush/main.py

```diff
@@ -134,6 +134,10 @@ def add_shared_args(parser):
     parser.add_argument('--router_write_throughput',
                         help="DynamoDB router write throughput",
                         type=int, default=5, env_var="ROUTER_WRITE_THROUGHPUT")
+    parser.add_argument('--router_timeout',
+                        help="Seconds to wait for a router to process a "
+                             "message",
```

Comment:
This isn't really timing out the entire router, is it? It's just changing the timeout on requests the router might make?